pkg/frontend/metrics.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/recover"
+)
+
+//emitMetrics sends a heartbeat metric every 60 seconds
+func (f *frontend) emitMetrics(stop <-chan struct{}) {
+	defer recover.Panic(f.baseLog)
+	t := time.NewTicker(time.Minute)
+	defer t.Stop()
+	f.baseLog.Print("starting heartbeat")
+
+	for {
+		select {
+		case <-t.C:
+			if f.checkReady() {
+				f.m.EmitGauge("frontend.healthz.ready", 1, map[string]string{
```

Comment:
This goes :man_shrugging:  in my head. Because in prometheus world we would have : 
```
frontend.healthz.ready 1 
frontend.healthz.ready 0
```
 This is confusing. I know the reason we doing this, but maybe worth discussing?

Reply:
Removed the else clause, only a "ready" heartbeat will add a datapoint now.